index.html

```diff
@@ -76,6 +76,8 @@ <h1>W3C on GitHub</h1>
       to automatically publish on every commit (to a given branch) and never again have to worry
       about what people used to call “heartbeat” publications. With this, you can even get rid of
       the notion of Editor's Draft altogether.
+      If your spec is generated using Bikeshed, check out
+      <a href="https://github.com/rtoy/auto-deploy-spec">how to auto-deploy Bikeshed specs</a>.
```

Comment:
IMHO, that new sentence about auto-deploying Bikeshed-built specs should be marked up a separate paragraph — so that it’s rendered with a line break before. Otherwise, I think it would be easy for readers to just overlook, since it’s just tacked on the preceding sentences in the paragraph — which are all entirely about publishing to /TR space with Echidna (which is different thing than what the https://github.com/rtoy/auto-deploy-spec guidelines are about.

Reply:
we should mention https://github.com/w3c/echidna/wiki/How-to-use-Echidna-with-Bikeshed-and-GitHub or move that wiki into w3c.github.io